examples/mod-data-import-converter-storage/matchProfileCollection.sample

```diff
@@ -17,10 +17,10 @@
         "incomingIndicator1": " ",
         "incomingIndicator2": " ",
         "incomingRecordSubfield": "p",
-        "incomingStaticValueType": "",
+        "incomingStaticValueType": null,
         "incomingText": "",
         "incomingNumber": "",
-        "incomingDateType": "",
+        "incomingDateType": null,
         "incomingFromDate": "",
```

Comment:
Does an empty string (for example "incomingNumber": "") appear in final response for sample data ?
or it will be flushed in response ?

Reply:
Bad idea to add nulls. Maybe add some change to marshaler or entity

Reply:
> Does an empty string (for example "incomingNumber": "") appear in final response for sample data ?
> or it will be flushed by json marsheller (the same as null values for enums) ?

empty string doesn't appear in response for fields with "number" type or "date-time" format